Classes/BEMCheckBoxGroup.h

```diff
@@ -0,0 +1,39 @@
+//
+//  BEMCheckBoxGroup.h
+//  CheckBox
+//
+//  Created by Cory Imdieke on 10/17/16.
+//  Copyright © 2016 Boris Emorine. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class BEMCheckBox;
+
+@interface BEMCheckBoxGroup : NSObject
+
+/** An array of check boxes in this group.
+ */
+@property (nonatomic, strong, nonnull, readonly) NSArray<BEMCheckBox *> *checkBoxes;
```

Comment:
I don't think so actually, we do use the order of boxes and auto-select the first one so order does matter.

We could potentially use an ordered set, if you would prefer that.